Adyen/Helpers/UIViewHelpers.swift

```diff
@@ -64,4 +64,12 @@ extension AdyenScope where Base: UIView {
         base.alpha = hidden ? 0 : 1
         base.adyen.updatePreferredContentSize()
     }
+
+    public var getMinimalSize: CGSize {
```

Comment:
Naming it without get would be more Swifty`minimalSize`

Reply:
👍